src/Date/CalendarEdit.tsx

```diff
@@ -165,11 +176,26 @@ function CalendarInputPure(
     const year = Number(date.slice(yearIndex, yearIndex + 4))
     const month = Number(date.slice(monthIndex, monthIndex + 2))
 
+    if (year % 400 === 0 || (year % 100 !== 0 && year % 4 === 0))
+      monthLength[1] = 29
```

Comment:
Why 29?

Reply:
Leap year 29 days. index 1 will be Feb month. Above if conditions is verify if it is leap year.